src/atomate2/aims/run.py

```diff
@@ -0,0 +1,113 @@
+"""An FHI-aims jobflow runner."""
+from __future__ import annotations
+
+import json
+import logging
+import os
+import subprocess
+from os.path import expandvars
+from typing import TYPE_CHECKING
+
+from ase.calculators.aims import Aims
+from ase.calculators.socketio import SocketIOCalculator
+from monty.json import MontyDecoder
+
+if TYPE_CHECKING:
+    from atomate2.aims.schemas.task import AimsTaskDoc
+    from atomate2.aims.utils.msonable_atoms import MSONableAtoms
+
+logger = logging.getLogger(__name__)
+
+
+def run_aims(
+    aims_cmd: str = None,
+):
+    """
+    Run FHI-aims.
+
+    Parameters
+    ----------
+    aims_cmd : str
+        The command used to run FHI-aims (defaults to ASE_AIMS_COMMAND env variable).
+    """
+    if aims_cmd is None:
+        aims_cmd = os.getenv("ASE_AIMS_COMMAND", "aims.x")
```

Comment:
What do you think about adding an AIMS_COMMAND option to the atomate2 settings, which is used in preference to the ASE_AIMS_COMMAND?

Reply:
Ya I can make that change